src/widgets/NavigationBar/ui/Notifications.tsx

```
@@ -29,28 +46,62 @@ export default function Notifications() {
             ))}
           </div>
-          <Link className="text-[#000]" href="/notifications">
-            전체보기
-          </Link>
+
+          <div className="p-4 border-t border-border">
+            <button
+              onClick={handleViewAll}
+              className="w-full bg-primary text-primary-foreground rounded-[10px] px-4 py-2 hover:bg-[#276e48] transition-all duration-200 hover:shadow-hover active:bg-[#1e3e2c] active:scale-[0.98]"
+            >
+              전체보기
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)